modules/nodes-pods-user-namespaces-configuring.adoc

```diff
@@ -0,0 +1,329 @@
+
+* After the nodes return to a `READY` state, enable the crun container runtime to use with new containers on the worker nodes:
++
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: ContainerRuntimeConfig
+metadata:
+ name: enable-crun-worker
+spec:
+ machineConfigPoolSelector:
+   matchLabels:
+     pools.operator.machineconfiguration.openshift.io/worker: "" <1>
+ containerRuntimeConfig:
+   defaultRuntime: crun <2>
+----
+<1> Specifies the machine config pool label. 
+<2> Specifies the container runtime to deploy to new containers. The default value is `runc`.
+
+////
+Not needed for TP
+
+. Enable rootless networking for nested containers:
+
+.. Create a Butane config file similar to the following example:
++
+[source,yaml,subs="attributes+"]
+----
+variant: openshift
+version: 4.17.0
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: nested-podman-worker
+storage:
+  files:
+  - path: /etc/crio/crio.conf.d/99-nested-podman
+    mode: 0644
+    overwrite: true
+    contents:
+      inline: |
+        [crio.runtime]
+        allowed_devices = [
+          "/dev/fuse",
+          "/dev/net/tun"
+        ]
+  - path: /etc/modules-load.d/nested-podman.conf
+    mode: 0644
+    overwrite: true
+    contents:
+      inline: |
+        tun
+        rtnl-link-bridge
+        ipt_addrtype
+        ipt_MASQUERADE
+  - path: /etc/nested_podman/nested_podman.te
+    mode: 0644
+    overwrite: true
+    contents:
+      inline: |
+        module nested_podman 1.0;
+
+        require {
+          type null_device_t;
+          type zero_device_t;
+          type urandom_device_t;
+          type random_device_t;
+          type container_file_t;
+          type container_engine_t;
+          type devtty_t;
+          type setfiles_t;
+          class chr_file { mounton read setattr write };
+          class sock_file mounton;
+        }
+
+        #============= container_engine_t ==============
+        allow container_engine_t container_file_t:sock_file mounton;
+        allow container_engine_t devtty_t:chr_file mounton;
+        allow container_engine_t null_device_t:chr_file { mounton setattr };
+        allow container_engine_t random_device_t:chr_file mounton;
+        allow container_engine_t urandom_device_t:chr_file mounton;
+        allow container_engine_t zero_device_t:chr_file mounton;
+systemd:
+  units:
+  - contents: |
+      [Unit]
+      Description=Modify SeLinux Type container_engine_t
+      DefaultDependencies=no
+      After=kubelet.service
+
+      [Service]
+      Type=oneshot
+      RemainAfterExit=yes
+      ExecStart=bash -c "/bin/checkmodule -M -m -o /tmp/nested_podman.mod /etc/nested_podman/nested_podman.te && /bin/semodule_package -o /tmp/nested_podman.pp -m /tmp/nested_podman.mod && /sbin/semodule -i /tmp/nested_podman.pp"
+      TimeoutSec=0
+
+      [Install]
+      WantedBy=multi-user.target
+    enabled: true
+    name: systemd-nested-podman-selinux.service
```

Comment:
I would say we can drop this as well. the aim is to include these in 4.17.1, and the selinux profile exists on 4.17.0 so it's usabe, just not fully complete

Reply:
Removed the section on nested container configuration.